src/components/Analysis/TabsNav.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useLocation, useHistory } from 'react-router-dom'
+
+import Paper from 'cozy-ui/transpiled/react/Paper'
+import { Tabs, Tab } from 'cozy-ui/transpiled/react/MuiTabs'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+const routes = ['/analysis/modes', '/analysis/purposes']
+
+const a11yProps = index => ({
+  id: `simple-tab-${index}`,
+  'aria-controls': `simple-tabpanel-${index}`
+})
+
+const TabsNav = () => {
+  const history = useHistory()
+  const { pathname } = useLocation()
+  const { t } = useI18n()
+
+  const handleChange = (_, value) => {
+    history.push(routes[value])
```

Comment:
that function is not tested :( 

Shoudn't we test it during development?

Reply:
vu ce qu'elle fait, est-ce bien nécessaire ? Quel effet de bord as-tu en tête qu'un test éviterait ?

Reply:
```
import React from 'react'
import { fireEvent, render } from '@testing-library/react'
import TabsNav from './TabsNav'
import { useHistory } from 'react-router-dom'
import { useI18n } from 'cozy-ui/transpiled/react/I18n'

const history = {
  push: x => {
    // eslint-disable-next-line no-console
    console.log('history push is called')
    return x
  }
}
jest.mock('react-router-dom', () => ({
  useHistory: jest.fn().mockReturnValue(history),
  useLocation: jest.fn().mockReturnValue('pathname')
}))
jest.mock('cozy-ui/transpiled/react/I18n')

jest.mock('cozy-ui/transpiled/react/MuiTabs', () => ({
  Tabs: ({ onChange, value }) => (
    <button onClick={onChange} data-testid="Tabs" data-value={value} />
  ),
  Tab: ({ label, children }) => (
    <div data-testid="Tab" data-label={label}>
      {children}
    </div>
  )
}))

describe('TabsNav', () => {
  const t = x => x

  beforeEach(() => {
    useI18n.mockReturnValue({ t })
    jest.spyOn(console, 'log').mockImplementation()
  })

  it('should push route', () => {
    // Given
    const { getByTestId } = render(<TabsNav />)
    useHistory.mockReturnValue(history)

    // When
    fireEvent.click(getByTestId('Tabs'))

    // Then
    // eslint-disable-next-line no-console
    expect(console.log).toHaveBeenCalledWith('history push is called')
  })
})
```